README.md

```diff
@@ -129,8 +131,25 @@ The first command downloads the virtual machine image, and then runs our provisi
     vm$ cargo build
     vm$ ./xtests/run
     All the tests passed!
-
-
+   
 ### Running without Vagrant
 
 Of course, the drawback of having a standard development environment is that you stop noticing bugs that occur outside of it. For this reason, Vagrant isn’t a *necessary* development step — it’s there if you’d like to use it, but exa still gets used and tested on other platforms. It can still be built and compiled on any target triple that it supports, VM or no VM, with `cargo build` and `cargo test`.
+
+### Ubuntu dependencies
+For installing on ubunutu18 try this:
+
+    apt install cmake
+    apt install libssl1.0-dev
+    apt install pkg-config
+    apt install cargo
```

Comment:
Maybe easyer to use one apt install (`apt install cmake libssl1.0-dev pkg-config cargo`).